deploy-manage/security/_snippets/cluster-comparison.md

```diff
@@ -17,70 +18,84 @@ Select your deployment type below to see what's available and how implementation
+| | Secure settings | Configurable | Automatically protected by Elastic |
+| | Saved object encryption | Fully managed | Automatically encrypted by Elastic |
+| **User Session** | Kibana Sessions | Configurable | [Customize session parameters](/deploy-manage/security/kibana-session-management.md) |
 
 :::
 
-:::{tab-item} {{serverless-full}}
+:::{tab-item} Serverless
 :sync: serverless
 
 | Category| Security feature | Status | Description |
 |------------------|------------|--------------|-------------|
-| **Communication** | TLS (HTTP Layer) | Managed | Automatically configured by Elastic |
-| | TLS (Transport Layer) | Managed | Automatically configured by Elastic |
-| **Network** | IP traffic filtering | Configurable | Configure IP-based access restrictions |
+| **Communication** | TLS (HTTP Layer) | Fully managed | Automatically configured by Elastic |
+| | TLS (Transport Layer) | Fully managed | Automatically configured by Elastic |
+| **Network** | IP traffic filtering | Configurable | [Configure IP-based access restrictions](/deploy-manage/security/ip-traffic-filtering.md) |
 | | Private link | N/A | X |
-| | Static IPs | Configurable | Enable fixed IP addresses |
-| **Data** | Encryption at rest | Managed | Automatically encrypted by Elastic |
-| | Bring your own encryption key | N/A | X |
-| | Keystore security | Managed | Automatically protected by Elastic |
-| | Saved object encryption | Managed | Automatically encrypted by Elastic |
+| | Kubernetes Network Policies | N/A |  |
+| **Data** | Encryption at rest | Fully managed | Automatically encrypted by Elastic |
+| | Secure settings | Configurable | Automatically protected by Elastic |
+| | Saved object encryption | Fully managed | Automatically encrypted by Elastic |
 | **User Session** | Kibana Sessions | Managed | Automatically configured by Elastic |
 
 :::
 
-:::{tab-item} ECE/ECK
-:sync: ece-eck
+:::{tab-item} ECE
+:sync: ece
 
 | Category| Security feature | Status | Description |
 |------------------|------------|--------------|-------------|
-| **Communication** | TLS (HTTP Layer) | Configurable | Configure custom certificates |
-| | TLS (Transport Layer) | Managed | Automatically configured by Elastic |
-| **Network** | IP traffic filtering | Configurable | Configure IP-based access restrictions |
+| **Communication** | TLS (HTTP Layer) | Managed | You can [configure custom certificates](/deploy-manage/security/secure-your-elastic-cloud-enterprise-installation/manage-security-certificates.md) |
+| | TLS (Transport Layer) | Fully managed | Automatically configured by Elastic |
+| **Network** | IP traffic filtering | Configurable | [Configure IP-based access restrictions](/deploy-manage/security/ip-traffic-filtering.md) |
 | | Private link | N/A | X |
-| | Static IPs | N/A | X |
+| | Kubernetes Network Policies | N/A |  |
 | **Data** | Encryption at rest | Self-managed | Implement at infrastructure level |
```

Comment:
careful with these infra level comments. we need to make it clear that this is not necessarily a supported configuration. perhaps use an annotation like a star or dagger